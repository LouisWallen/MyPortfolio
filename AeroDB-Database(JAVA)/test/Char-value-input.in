set a 1 2 3
set b 5 6
snapshot
list ENTRIES
set c 5.5
list entries
set a 5r 4.7 r
list entRies
push a 7.7 t
set c 8 9 3
append c 9.9 p
snapshot
list entries
pic c 5
pick c 2
pick c 2r
get c
pluck a 1.0
pluck a 1
get a
drop 9.0
list SNAPSHOTS
rollback 1t
rollback 1
list SNAPSHOTS
set d 9 34 2
snapshot
list snapshots
checkout 2.1
list entRies
bye
