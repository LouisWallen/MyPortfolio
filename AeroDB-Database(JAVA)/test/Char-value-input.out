> ok

> ok

> saved as snapshot 1

> b [5 6]
a [1 2 3]

> Please input integers in value of the key

> b [5 6]
a [1 2 3]

> Please input integers in value of the key

> b [5 6]
a [1 2 3]

> Please input integers in value of the key

> ok

> Please input integers in value of the key

> saved as snapshot 2

> c [8 9 3]
b [5 6]
a [1 2 3]

> Wrong Command!

> 9

> Please input integers in index of the key

> [8 9 3]

> Please input integers in index of the key

> 1

> [2 3]

> Please input integers in id of the snapshot

> 2
1

> Please input integers in id of the snapshot

> ok

> 1

> ok

> saved as snapshot 2

> 2
1

> Please input integers in id of the snapshot

> ok

> d [9 34 2]
b [5 6]
a [2 3]

> bye
